crates/sui-types/src/gas.rs

```diff
@@ -75,13 +80,16 @@ impl StorageCost {
 
 /// A list of constant costs of various operations in Sui.
 struct SuiCostTable {
-    /// A flat fee charged for every transaction. This is also the mimmum amount of
-    /// gas charged for a transaction.
+    /// A flat fee charged for every transaction. Independent of size.
     pub min_transaction_cost: ComputationCost,
+
+    /// This gets charged for each byte of the transaction
+    pub transaction_cost_per_byte: ComputationCost,
```

Comment:
I think we can just use `transaction_cost_per_byte` to trump `min_transaction_cost`. Not much need to have two params

Reply:
@lxfind I recommend reading & commenting on the doc that outlines all these params because I assumed we had settled on what was going in. The `min_transaction_cost` covers the base fee for accepting to operate on this TX, and for other constants we're not charging for in the size dependent fee.

Reply:
I would agree with @lxfind in that it feels as if `min_transaction_cost` is softly determined  by the transaction cost per byte, as there is inherently a minimum transaction size